components/cli/pkg/util/utils.go

```diff
@@ -904,14 +900,12 @@ func PrintSuccessMessage(message string) {
 func GetSourceFileName(filePath string) (string, error) {
 	d, err := os.Open(filePath)
 	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+		ExitWithErrorMessage("Error opening file", err)
```

Comment:
Shall we add the filepath to the error message? Fix in other places as well

Reply:
Updated